pkg/relayer/proxy/proxy.go

```diff
@@ -187,3 +191,25 @@ func (rp *relayerProxy) validateConfig() error {
 
 	return nil
 }
+
+// Ping tests the connectivity between all the managed relay servers and their respective backend URLs.
+func (rp *relayerProxy) Ping(ctx context.Context) []error {
+	var errs []error
+
+	var i int
+	for _, srv := range rp.servers {
+		if err := srv.Ping(ctx); err != nil {
+			rp.logger.Error().Err(err).
+				Msg("an unexpected error occured while pinging backend URL")
+			errs = append(errs, err)
+		}
+
+		i++
+	}
```

Comment:
```suggestion
	for i, srv := range rp.servers {
		if err := srv.Ping(ctx); err != nil {
			rp.logger.Error().Err(err).
				Msg("an unexpected error occured while pinging backend URL")
			errs = append(errs, err)
		}
	}
```

Reply:
Nice catch. I forgot to do some cleanup here. 🙏🏾